des_y6utils/mdet.py

```diff
@@ -514,11 +631,98 @@ def _make_mdet_cuts_raw_v345(
         & (mag_r < 26.5)
         & (mag_i < 26.2)
         & (mag_z < 25.6)
-        & (d["pgauss_T"] < (1.2 - 3.1*d["pgauss_T_err"]))
```

Comment:
Is this a bug that means all of our tests with this never actually did anything?